### DISCUSSION TITLE
what's the difference between these two?

### DISCUSSION THREAD

**Discussion by hisengberg** [anujpathania/HotSniper#33] [Q&A] [answered]
![image](https://github.com/anujpathania/HotSniper/assets/118739669/2ed7abce-f41f-4625-84d7-baa4ac4598ae)

As shown above, what do these two terms mean? and what's the difference between them?

I would be very grateful if anyone could help me.

**Comment by anujpathania** [comment]
Hi,

This questions should be posted to Sniper Devs, as this code is inherited from them in HotSniper. You can post it in their Google Forums - https://groups.google.com/g/snipersim

The closest hit I can get on this question in Sniper forum is this existing discussion - https://groups.google.com/g/snipersim/c/QgTJdU55CZE

**Reply by hisengberg** [reply, marked as answer]
Hi anuj,

Thinks for your reply, i will go to Sniper Devs and take a look.
